src/xml_processing/query_pubmed_mod_updates.py

```diff
@@ -152,7 +152,7 @@ def get_pmid_association_to_mod_via_reference(pmids: List[str], mod_abbreviation
         ReferenceModel.curie,
         ModModel.abbreviation
     ).join(
-        ReferenceModel.cross_references
+        ReferenceModel.cross_reference
```

Comment:
Oh, I'm surprised this wasn't part of the fixes Ian did when changing it from plural to singular, thanks for catching this !